clustering/web/hotrod/src/main/resources/org.wildfly.clustering.web.hotrod.session.proto

```diff
@@ -0,0 +1,15 @@
+package org.wildfly.clustering.web.hotrod.session;
+
+/**
+ * @TypeId(210)
```

Comment:
This doesn't seem right – the TypeId is the same as in `src/main/resources/org.wildfly.clustering.web.infinispan.session.proto` yet the message type is differnet. Is that intentional?

Reply:
Yes, this is intentional.  These types are mutually exclusive.